src/containerapp/azext_containerapp/containerapp_resiliency_decorator.py

```diff
@@ -0,0 +1,476 @@
+            timeout_policy['responseTimeoutInSeconds'] = timeout_policy.get('responseTimeoutInSeconds', DEFAULT_RESPONSE_TIMEOUT)
+            timeout_policy['connectionTimeoutInSeconds'] = timeout_policy.get('connectionTimeoutInSeconds', DEFAULT_CONNECTION_TIMEOUT)
+            containerapp_def['properties']['timeoutPolicy'] = timeout_policy
+
+        # Circuit Breaker
+        circuit_breaker_policy = safe_get(containerapp_def, 'properties', 'circuitBreakerPolicy')
+        if circuit_breaker_policy and ('consecutiveErrors' in circuit_breaker_policy or 'intervalInSeconds' in circuit_breaker_policy or 'maxEjectionPercent' in circuit_breaker_policy):
+            circuit_breaker_policy['consecutiveErrors'] = circuit_breaker_policy.get('consecutiveErrors', DEFAULT_CONSECUTIVE_ERRORS)
+            circuit_breaker_policy['intervalInSeconds'] = circuit_breaker_policy.get('intervalInSeconds', DEFAULT_INTERVAL)
+            circuit_breaker_policy['maxEjectionPercent'] = circuit_breaker_policy.get('maxEjectionPercent', DEFAULT_MAX_EJECTION)
+            containerapp_def['properties']['circuitBreakerPolicy'] = circuit_breaker_policy
+
+        # HTTP Connection Pool
+        http_connection_pool = safe_get(containerapp_def, 'properties', 'httpConnectionPool')
+        if http_connection_pool and ('http1MaxPendingRequests' in http_connection_pool or 'http2MaxRequests' in http_connection_pool):
+            http_connection_pool['http1MaxPendingRequests'] = http_connection_pool.get('http1MaxPendingRequests', DEFAULT_HTTP1_MAX_PENDING_REQ)
+            http_connection_pool['http2MaxRequests'] = http_connection_pool.get('http2MaxRequests', DEFAULT_HTTP2_MAX_REQ)
+            containerapp_def['properties']['httpConnectionPool'] = http_connection_pool
+
+        return containerapp_def
+
+
+class ContainerAppResiliencyPreviewCreateDecorator(ContainerAppResiliencyDecorator):
+    def __init__(self, cmd: AzCliCommand, client: Any, raw_parameters: Dict, models: str):
+        super().__init__(cmd, client, raw_parameters, models)
+        self.containerapp_resiliency_def = ContainerAppsResiliencyModel
+
+    def create(self):
+        try:
+            r = self.client.create_or_update(
+                cmd=self.cmd, resource_group_name=self.get_argument_resource_group_name(),
+                name=self.get_argument_name(), container_app_name=self.get_argument_container_app_name(),
+                container_app_resiliency_envelope=self.containerapp_resiliency_def,
+                no_wait=self.get_argument_no_wait())
+            r = clean_null_values(r)
+            return r
+        except Exception as e:
+            handle_raw_exception(e)
+
+    def construct_payload(self):
+        if self.get_argument_yaml():
+            self.containerapp_resiliency_def = self.set_up_containerapp_resiliency_yaml(file_name=self.get_argument_yaml())
+            return
+
+        if self.get_argument_default():
```

Comment:
I want to know, if command without --default, will the default value for argument be used? For example, 
if I didn't enable `--default`, what is the value used for `timeout_response_in_seconds`?
`c.argument('timeout_response_in_seconds', type=int, options_list=['--timeout'], help='Specifies the timeout in seconds. This spans between the point at which the entire request has been processed and when the response has been completely processed. This timeout includes all retries. Default: 30.')`

Reply:
Just want to make clear what scenario `--default` work for.

Reply:
As you reply in https://github.com/Azure/azure-cli-extensions/pull/6967#discussion_r1387455732
The default value in the description message only be used when a particular property of a policy is set.
Whether it will cause confusion to users ， I cannot be seen get the meaning of  "only be used when a particular property of a policy is set " from the description in the `help`.

Reply:
I thought it will be too cluttered in the cli help because I will have to mention it for all commands. The docs will mention that part.